src/dcd_mapping/annotate.py

```diff
@@ -287,24 +288,27 @@ def _annotate_allele_mapping(
         mavedb_id=mapped_score.accession_id,
         score=float(mapped_score.score) if mapped_score.score else None,
         annotation_layer=mapped_score.annotation_layer,
+        error_message=mapped_score.error_message
+        if mapped_score.error_message
+        else None,  # TODO might not need if statement here
```

Comment:
Based on the way the model is defined, I agree you shouldn't need the if statement.